cpp-client/deephaven/examples/concurrent_client/main.cc

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+ */
+#include <exception>
+#include <iostream>
+#include <iomanip>
+#include <memory>
+#include <cstdlib>
+#include <thread>
+#include "deephaven/client/client.h"
+
+using deephaven::client::Client;
+using deephaven::client::TableHandleManager;
+
+void Usage(const char *const argv0, const int exit_status) {
+  std::cerr << "Usage: " << argv0 << "[numthreads [iterations [host:port]]]" << '\n';
+  std::exit(exit_status);
+}
+
+void ThreadFun(std::shared_ptr<Client> clientp, const std::size_t ti) {
+    auto manager = clientp->GetManager();
+    using namespace std::chrono_literals;
+    std::cout << "THREAD START " << ti << std::endl << std::flush;
+    const std::string t1_name = std::string("import deephaven; t1_") + std::to_string(ti);
+    while (true) {
+        manager.RunScript(
+                t1_name + "= deephaven.time_table(\"PT1S\")");
+        std::this_thread::sleep_for(2000ms);
+        auto t1 = manager.FetchTable(std::string("t1_") + std::to_string(ti));
+    }
+    std::cout << "THREAD END " << ti << std::endl << std::flush;
+}
+
+void Run(std::shared_ptr<Client> clientp, const std::size_t nthreads) {
+  std::thread threads[nthreads];
+  for (std::size_t i = 0; i < nthreads; ++i) {
+    threads[i] = std::move(std::thread(&ThreadFun, clientp, i));
```

Comment:
you shouldn't need std::move() here because you're working with a temporary.

Anyway if you switch to vector I believe this could become

```
threads.emplace_back(&ThreadFun, clientp, i);
```